src/OrbitQt/orbitsamplingreport.ui

```diff
@@ -103,7 +103,7 @@
                <string/>
               </property>
               <property name="icon">
-               <iconset resource="../icons/orbiticons.qrc">
+               <iconset>
```

Comment:
I guess this is not intended? I'm wondering if this has unexpected effects.

Reply:
Arg. All changes in this file were done by QtDesigner... I'll revert them all.